modules/javafx.web/src/test/java/test/javafx/scene/web/HTMLEditingTest.java

```diff
@@ -72,13 +86,14 @@
                             false, PlatformUtil.isMac()));// Cmd+V (Mac)
 
             assertEquals("Source Default value",getEngine().
-                    executeScript("srcInput.defaultValue").toString(),
-                    defaultText);
-            assertEquals("Source clipboard onpaste data", getEngine().
-                    executeScript("srcInput.value").toString(), clipboardData + defaultText);
+                    executeScript("srcInput.defaultValue").toString(), defaultText);
+            assertEquals("Source clipboard onpaste data", getEngine().executeScript("srcInput.value").
+                    toString(), clipboardData + defaultText);
```

Comment:
The expected and actual values are backwards; the first of the two objects being compared should be the expected value. I see that this is a preexisting bug in the test. I recommend fixing this in the new test (while leaving the existing test method as is).